invenio_app_ils/config.py

```diff
@@ -192,6 +196,10 @@ def _(x):
 MAIL_NOTIFY_CC = []
 #: Email BCC address(es) for email notification.
 MAIL_NOTIFY_BCC = []
+# Enable sending mail to test recipients.
+ENABLE_TEST_RECIPIENTS = True
+#: When variable ENABLE_TEST_RECIPIENTS is True all emails are send here.
+MAIL_NOTIFY_TEST_RECIPIENTS = ['info@inveniosoftware.org']
```

Comment:
maybe something safer as an address because we will start to send on invenio official e-mail :D

Reply:
@kprzerwa I used the inveniosoftware email so we override it in the cds layer with the cds related stuff :)